docs/topics/request-response.rst

```diff
@@ -342,6 +343,13 @@ download_fail_on_dataloss
 Whether or not to fail on broken responses. See:
 :setting:`DOWNLOAD_FAIL_ON_DATALOSS`.
 
+.. reqmeta:: max_retry_times
+
+max_retry_times
+---------------
+
+The meta key is used set retry times per request. When initialized, the :setting:`RETRY_TIMES` setting will be ignored by the downloader middleware.
```

Comment:
I find the phrasing saying that the setting is being ignored a bit confusing.
I would suggest saying the meta key takes precedence over the setting.
Also, try using shorter lines (around 80 chars), they will be formatted nicely by Sphinx.